tests/DiContainer/Constructor/DiContainerAddDefinitionThroughConstructorTest.php

```diff
@@ -25,10 +25,13 @@ class DiContainerAddDefinitionThroughConstructorTest extends TestCase
      */
     public function testDefinitionWithoutStringIdentifier(iterable $definition): void
     {
-        $this->expectException(DiDefinitionExceptionInterface::class);
-        $this->expectExceptionMessage('must be a non-empty string');
-
-        new DiContainer($definition);
+        try {
+            new DiContainer($definition);
+        } catch (ContainerIdentifierExceptionInterface $e) {
+            self::assertInstanceOf(ContainerIdentifierExceptionInterface::class, $e);
+            self::assertStringContainsString('Definition identifier must be a non-empty string', $e->getMessage());
+            self::assertIsInt($e->getIdentifier());
+        }
```

Comment:
```suggestion
        $this->expectException(ContainerIdentifierExceptionInterface::class);
        $this->expectExceptionMessage('Definition identifier must be a non-empty string');
```
и где-то отдельно тогда проверить сообщение, сделать тест на message и identifier для `ContainerIdentifierException`